migration/migration/002-utils.sql

```diff
@@ -8,6 +8,7 @@ CREATE TABLE _prom_catalog.remote_commands(
     transactional BOOLEAN,
     command TEXT
 );
+GRANT ALL ON SEQUENCE _prom_catalog.remote_commands_seq_seq TO current_user;
```

Comment:
isn't current_user here the `postgres` superuser? Why do we need a grant to a superuser?

Reply:
This is a bizarre one that I don't fully understand. The upgrade path makes this sequence owned by super user as it does with other sequences. Oddly, in the fresh install path, this sequence somehow has no owner listed in the snapshot. So the upgrade tests were failing. All other sequences worked as expected. Adding this explicit ownership fixed it but I still don't understand why it was needed